saleor/graphql/attribute/types.py

```diff
@@ -94,6 +98,11 @@ def prepare_reference(attribute):
             .then(prepare_reference)
         )
 
+    @staticmethod
+    @traced_resolver
```

Comment:
IMO, no need to add `traced_resolver` here, looks like simple resolver. @fowczarek  Can you confirm?

Reply:
Yep. We should use `traced_resolver` only for complex resolvers.